source/elens.f90

```diff
@@ -482,7 +486,7 @@ subroutine parseChebyFile(ifile)
       goto 30
     end if
     call chr_cast(lnSplit(3),tmpflt,spErr)
-    gamma = tmpflt*c1m3/pmae+1 ! from kinetic energy
+    gamma = tmpflt*c1m3/pmae+one ! from kinetic energy
```

Comment:
This line needs brackets for consistent rounding.

Reply:
done - I will anyway review the entire code of elenses to make sure that there are no other left-overs of this type